tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/UpdateClaimedEpicsHandlerTests.cs

```diff
@@ -55,7 +75,7 @@ public void ShouldThrowNotFoundExceptionWhenEpicsInfoIsNull(string epicId, strin
                 Mock.Of<IClaimedEpic>(e => e.EpicId == epicId && e.StatusName == statusName)
             };
 
-            Assert.ThrowsAsync<NotFoundException>(() => UpdateClaimedEpicsAsync(InvalidSolutionId, claimedEpics));
+            Assert.ThrowsAsync<ArgumentNullException>(() => UpdateClaimedEpicsAsync(ValidSolutionId, claimedEpics));
```

Comment:
Why would passing a valid solution id cause an exception or is it the list that is invalid?